### ISSUE TITLE
Training settings

### ISSUE THREAD

**Issue by Kai-Qi** [sword98/CKD-TransBTS#15]
Great work! I would love to cite your paper. 
Besides adjusting the batch size, are there any other techniques that could help me reproduce the results from your paper? I followed the settings from both the GitHub repository and the paper, but the Dice accuracy of the model I ran is 15 points lower than the reported results. 
Looking forward to your reply. Thank you !

**Comment by Yinaysa** [comment]
我现在也是这个问题，请问问题现在解决了吗？
期待您的回复！非常感谢！
